Extract calculation from formatting

avegancafe opened the pull request:
I thought it'd be funny to show this in my lualine, so I needed to extract this method to change the text formatting

avegancafe commented on plugin/configpulse.vim:
```diff
@@ -0,0 +1 @@
+command! ConfigPulse lua require"configpulse".find_time() 
```
This file gets loaded automatically by neovim, so you don't have to `require('configpulse')` first to get the command to register. I can remove if you don't want it though

mrquantumcodes commented:
I didn't really get your edits and there are conflicts I don't know how to resolve (sorry, I am not an expert). But I have updated the plugin so now when you use use the `require('configpulse').find_time(true)` method you'll get a table with the days, hours and minutes since modification. (You need to pass in the `true` parameter otherwise it will simply print it out).

avegancafe commented:
@mrquantumcodes are you still open to this? I just fixed the merge conflicts. This was defining global variables and the methodology was awkward so I refactored and fixed the issues, as well as making the command registration automatic so you don't have to require the lua module before using `:ConfigPulse`. If this isn't merged I can just use my own fork too

avegancafe commented:
I basically made the stuff between your `should_return` checks into its own function `get_time` and then swapped the implementation to use that. I'm not using `find_time` personally but this is backwards-compatible

mrquantumcodes commented:
Done, thanks for your contribution!
